readme.md

```diff
@@ -27,6 +27,7 @@ Home Assistant](https://www.awesome-ha.com/) initiée et hebergé par @frenck._
 - [Projets DIY](https://projetsdiy.fr) - Plein de guide DIY avec beaucoup d'ESP, d'électronique, des raspberry et de la domotique.
 - [&lt;DEVOTICS/&gt;](https://devotics.fr/) - Blog domotique, objets connectés, nouvelles technologies, développement.
 - [figer](https://www.figer.com/) - Blog de guide et conseil en informatique, réseau et domotique.
+- [Tech Café](https://techcafe.fr/category/domotique-podcast-maison-connectee/) - Un podcast domotique généraliste avec une sensibilité Jeedom mais qui s'ouvre à HA avec un épisode spécial HA à venir.
```

Comment:
```suggestion
- [Tech Café](https://techcafe.fr/category/domotique-podcast-maison-connectee/) - Un podcast domotique généraliste avec une sensibilité Jeedom mais qui s'ouvre à Home Assistant.
```
The incoming podcast episode information will be soon deprecated. 
I suggest we wait the episode being published and we will be able to modify the link description to advise on listening this specific episode.

Reply:
You're right, I pasted it too quickly whithout review the text.

Reply:
The best practice is you commit the suggestion I've made to let you have the final word on the modification in your PR.
Currently the suggestion I've made is not implemented.